<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsResultWidget</class>
 <widget class="QWidget" name="DetailsResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="nameField"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Surname</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="surnameField"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="positionField"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>City</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="cityFieldResult"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>University</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="univercityFieldResult"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLineEdit" name="phoneField"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLineEdit" name="emailField"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Skype</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLineEdit" name="skypeField"/>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string> Profile Url</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLineEdit" name="urlField"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
